src/App.js

```diff
@@ -1,26 +1,103 @@
+// Library
 import React from 'react';
-import logo from './logo.svg';
+// Styling
+import 'bulma/css/bulma.css';
 import './App.css';
+// File import
+import foods from './foods.json';
+import FoodList from './components/FoodList';
+import Search from './components/Search';
+import AddFood from './components/AddFood';
+import TodaysFood from './components/TodaysFood';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  state = {
+    foods: foods,
+    filteredFoods: foods,
+    addingFood: false,
+    todaysFood:[]
+  }
+
+  handleAddForm = () => {
+    console.log("add food clicked")
+    this.setState({
+      addingFood: !this.state.addingFood
+    })
+  }
+
+  handleAddFood = (event) => {
+    event.preventDefault()
+    let newItem = {
+      name: event.currentTarget.name.value,
+      image: event.currentTarget.image.value,
+      calories: event.currentTarget.calories.value,
+      quantity: 0
+    }
+    this.setState({
+      foods: [newItem, ...this.state.foods],
+      filteredFoods: [newItem, ...this.state.foods],
+      addingFood: false
+    })
+    event.currentTarget.name.value = ""
```

Comment:
these 3 lines are not needed. The input fields from AddFood are being automatically reset because the component is being rendered again with no initial values set.